Expanded docker.md documentation (#1802)
umap-project/umap: docs/config/icons.md

## Import icons automatically

> To import icons on your uMap server, you will need to use the command `umap import_pictograms` from inside main project folder.

Reviewer: ```suggestion
To import icons on your uMap server, you will need to use the command `umap import_pictograms` (making sure the virtualenv is active).
```
